src-v2/NTS.Domain.Setup/Entities/Contestant.cs

```diff
@@ -42,7 +42,7 @@ public override string ToString()
     {
         var sb = new StringBuilder();
         var startTimeMessage = StartTimeOverride == null ? "" : "StartTime: " + $"{StartTimeOverride.Value.ToLocalTime().TimeOfDay}";
-        var IsUnrankedMessage = IsUnranked ? "Unranked: Yes" : "Unranked: No";
+        var IsUnrankedMessage = IsUnranked ? "Unranked" : "Ranked";
         sb.Append($"Contestant -> Tandem: *isn't configured yet* {startTimeMessage} {IsUnrankedMessage} ");
```

Comment:
If `startTimeMessage`  is empty here it would lead to duplicate spaces, right? Isn't there a better solution?